### PR TITLE
Update build-and-upload.yml for Azure Pipelines

### PR DESCRIPTION
**Pull request by RuoyuWang-MS**

### CODE REVIEW

**Review by Flanker32 — DISMISSED** [review]
LGTM

File: .azure-pipelines/build-and-upload.yml [hunk 1]
```diff
@@ -63,6 +63,7 @@ extends:
                     BuildWebApp=$(BUILD_WEB_APP)
                     BuildSpringCloud=$(BUILD_SPRING_CLOUD)
                     BuildSpringApps=$(BUILD_SPRING_APPS)
+                    BuildContainerApps=$(BUILD_Container_APPS)
```
Comment (wangmingliang-ms) [inline .azure-pipelines/build-and-upload.yml]: why not all upper case?
`BUILD_Container_APPS` -> `BUILD_CONTAINER_APPS`

Comment (RuoyuWang-MS) [reply .azure-pipelines/build-and-upload.yml]: Fixed

**Review by wangmingliang-ms — APPROVED** [review]
LGTM

**Pull request merged**